lua/astrocommunity/media/codesnap-nvim/init.lua

```diff
@@ -20,6 +20,7 @@ return {
     },
   },
   opts = {
+    save_path = "~/Pictures",
```

Comment:
This should rather use `os.getenv( "HOME" )`

Reply:
can i know why? doesn't tilda rely on the HOME env internally?
or you just mean we should scrap the Pictures folder part?

Reply:
Not sure if `~` get resolved correctly on windows.

Reply:
oh for that actually this plugin doesn't support windows at all unless you use a very early stage version
so no worries in that regard

**But I'm working on it to use the correct version based on the system you have in a different branch i will make a pr when i tested it**

Reply:
Hi guys @Std-Enigma @Uzaaft , I just noticed this thread. 

CodeSnap has already parsed `~` and `$HOME` to the `$HOME` environment value in the lower level, thus in Lua level, it can pass `~` or `$HOME` directly, these values should be parse correctly by `generator` module of CodeSnap.nvim.

![image](https://github.com/user-attachments/assets/5e604d11-1202-42fe-9acd-0af83c3441f3)

Reply:
The discussion wasn't only limited to this plugin, but also for astrocommunity as a whole.